docs/react-native/connecting.mdx

```diff
@@ -11,22 +11,22 @@ This article will guide you through the process of connecting to a Fishjam room.
 
 ## Getting URL and token
 
-In order to connect, you need to retrieve `FISHJAM_URL` (multimedia server address) and `PEER_TOKEN` (token that will authenticate the peer in
+In order to connect, you need to retrieve **Fishjam URL** and **Peer Token** (token that will authenticate the peer in
 your Room).
 
 <Tabs groupId="app-type">
 
   <TabItem value="sandbox" label="Sandbox App">
 
 Once you get your account on [Fishjam](https://fishjam.io), you will have access to Sandbox App. This app comes
-with pre-configured test service called Room Manager. This is basically service that will create Room, add your app as
+with pre-configured test service called [Room Manager](/room-manager). This is basically service that will create Room, add your app as
 Room's Peer and return token required to use that Room.
 
 To use that simply call `fetch`:
 
 ```ts
 const response = await fetch(
-  `https://fishjam.io/api/v1/connect/${YOUR_ID}/room-manager/?roomName=${roomName}&peerName=${peerName}`,
+  `https://fishjam.io/api/v1/connect/${YOUR_APP_UUID}/room-manager/?roomName=${roomName}&peerName=${peerName}`,
```

Comment:
In room manager docs we use `YOUR_APP_UUID`, so let's have same name here

Reply:
Though this is inconsistent with the reality. The Room Manager UUID is different from the actual app ID. But I guess this is fine for simplicity’s sake?